src/xpk/parser/batch.py

```diff
@@ -36,6 +38,13 @@ def set_batch_parser(batch_parser):
       default=None,
       help='Cluster to which command applies.',
   )
+  batch_optional_arguments.add_argument(
+      '--local-test',
```

Comment:
```suggestion
      '--kind-cluster',
```
Maybe like this? And also name of cluster.

Reply:
Done.